### PR TITLE
Support test generators

### PR DESCRIPTION
**Pull request by bersace**
Fixes:

```
    Traceback (most recent call last):
      File "/usr/lib/python2.7/unittest/case.py", line 332, in run
        testMethod()
      File "/usr/local/lib/python2.7/site-packages/nose/loader.py", line 523, in makeTest
        return self._makeTest(obj, parent)
      File "/usr/local/lib/python2.7/site-packages/nose/loader.py", line 587, in _makeTest
        return self.loadTestsFromGenerator(obj, parent)
      File "/usr/local/lib/python2.7/site-packages/nose/loader.py", line 262, in loadTestsFromGenerator
        return self.suiteClass(generate, context=generator, can_split=False)
    TypeError: hackSuiteClass() got an unexpected keyword argument 'context'
```

### CODE REVIEW

**Comment by bersace** [comment]
By the way, congratulations for nose-randomly, very effective.

**Comment by adamchainz** [comment]
LGTM. Can you add a line to `HISTORY` above 1.2.0, and your details in AUTHORS - if you want that.

**Comment by bersace** [comment]
Updated. Thanks for the review !

**Comment by bersace** [comment]
:up:

File: HISTORY.rst [hunk 1]
```diff
@@ -3,6 +3,11 @@
 History
 -------
 
+1.2.0.dev0 (unreleased)
```
Comment (bersace) [inline HISTORY.rst:6]: Not sure about this :/

Comment (adamchainz) [inline HISTORY.rst:6]: Don't worry I'll update it

**Comment by adamchainz** [comment]
Merged in 3c97ed03c4fa5756c73bb30a2acb9d35cd1c23a9, thanks!

**Comment by adamchainz** [comment]
[Released on PyPI](https://pypi.python.org/pypi/nose-randomly)

**Comment by bersace** [comment]
Thanks for the release !!

**Pull request closed without merging**